clang/docs/ClangSYCLLinkWrapper.rst

```diff
@@ -0,0 +1,67 @@
+====================
+Clang SYCL link Wrapper
+====================
+
+.. contents::
+   :local:
+
+.. _clang-sycl-link-wrapper:
+
+Introduction
+============
+
+This tool works as a wrapper around the SYCL device code linking process.
+Purpose of this wrapper is to provide an interface to link SYCL device bitcode
```

Comment:
According to my understanding, the purpose of the tool is to provide standard linker interface and be able to link any object file formats 'compile' step is able to produce for SYCL offload mode. So, I would expect this tool to handle linking of fat objects with device code in other IR like SPIR-V as well as native binary format.

Reply:
Hi @bader 

Thanks for the feedback. I agree with you. This tool is expected to handle linking of fat objects and SPIR-V objects. However, in its current state, it can support only bitcode files. We can extend this tool easily to support linking of other objects. I was planning to update the document as the tool evolves. I will add a statement in the doc.

Reply:
Thanks! Let's clearly state the final goal for the tool, current status of the implementation, and plans for the future work.